src/lib/sedlexing.mli

```diff
@@ -136,6 +148,9 @@ val backtrack: lexbuf -> int
     internal slot of the buffer, and performs backtracking
     (the current position is set to the value of the backtrack position). *)
 
+val convert_for_menhir: (lexbuf -> 'token) -> lexbuf -> (unit -> 'token * Lexing.position * Lexing.position)
+(** [backtrack convert_for_menhir] convert a lexer to a rivised lexer that
+    menhirLib accepts *)
```

Comment:
I think a better (and more neutral) name would with `with_tokenizer`. The documentation could look like

```
[with_tokenizer tokenizer lexbuf] returns a generator of tokens annotated with positions. 
This generator can be used with {!TheRightMenhirFunction}
```